tools/proofers/hiero/index.php

```diff
@@ -2,15 +2,222 @@
+// as published by the Free Software Foundation; either version 2
+// of the License, or any later version.
+// 
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+// 
+// You should have received a copy of the GNU General Public License
+// along with this program; if not, write to the Free Software
+// Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA  02111-1307, USA.
+//
+//////////////////////////////////////////////////////////////////////////
+$lang=short_lang_code();
+$wh_img_url="$wikihiero_url/".WH_IMG_DIR;
+if($dh = opendir("$wikihiero_dir/".WH_IMG_DIR)) 
+{
+    while(($file = readdir($dh)) !== false) 
+    {
+        $files[]=$file;
+    }
+    closedir($dh);
+}
+natsort($files);
+foreach($files as $file) {
+    $code=WH_GetCode($file);
+    if($table == "All")
+    {
+        if(in_array($code, $wh_phonemes))
+            echo img("$wh_img_url/$file","$code [".array_search($code, $wh_phonemes)."]");
+        else
+            echo img("$wh_img_url/$file",$code);
+    }
+    else if($table == "Phoneme")
+    {
+        if(in_array($code, $wh_phonemes))
+            echo img("$wh_img_url/$file","$code [".array_search($code, $wh_phonemes)."]");
+    }
+    else if($table == "Aa")
+    {
+        if((substr($code, 0, 2) == $table) && ctype_digit($code[2]))
+        {
+            if(in_array($code, $wh_phonemes))
+                echo img("$wh_img_url/$file","$code [".array_search($code, $wh_phonemes)."]");
+            else
+                echo img("$wh_img_url/$file",$code);
+        }
+    }
+    else
+    {
+        if(isset($code[0]) && $code[0] == $table && ctype_digit($code[1]))
+        {
+            if(in_array($code, $wh_phonemes))
+                echo img("$wh_img_url/$file","$code [".array_search($code, $wh_phonemes)."]");
+            else
+                echo img("$wh_img_url/$file",$code);
+        }
+    }
+}
+
+function img($src,$title)
+{
+    $src = attr_safe($src);
+    $title = attr_safe($title);
+    $onClick = "add('". attr_safe(preg_replace(array("/^.*[[]/","/[]].*$/"), "" , $title)) . "');";
+    return "<img src='$src' title='$title' onClick=\"$onClick\">";
+}
+
+function table_options()
+{
+    global $tables, $table;
+    $table_options = "<option>".html_safe(WH_Text("Tables"))."</option>";
+    $table_options .= "<option>----</option>";
+    foreach($tables as $v) {
+        $table_options .= "<option value='" . attr_safe($v) . "'";
+        if ($v == $table) {
+            $table_options .= " selected";
+        }
+        $table_options .= ">";
+        if (strlen($v) <= 2)
+            $table_options .= html_safe($v) . " &mdash; ";
+        $table_options .= html_safe(WH_Text($v));
+        $table_options .= "</option>";
+    }
+
+    return $table_options;
+}
+
+function syntax_options()
```

Comment:
`syntax_options()` includes the `<select>` tags but `table_options()` doesn't. They should behave similarly. From the naming of them I would only expect them to include the `<option>`s.